src/minute/Snakefile

```diff
@@ -278,6 +289,19 @@ rule bowtie2:
         "| samtools sort -@ {threads} -o {output.bam} -"
 
 
+rule quality_filter_reads:
+    output:
+        bam=temp("tmp/5-mapq-filtered/{sample}_rep{replicate}.{reference}.bam")
+    input:
+        bam="tmp/4-mapped/{sample}_rep{replicate}.{reference}.bam"
+    run:
+        mapq = config.get("mapping_quality", 0)
+        if mapq == 0:
+            os.link(input.bam, output.bam)
+        else:
+            shell("samtools view -h -b {input.bam} -q {mapq} > {output.bam}")
```

Comment:
```suggestion
            shell("samtools view -q {mapq} -o {output.bam} {input.bam}")
```
Nowadays, samtools can detect the output format automatically if you provide the file name using `-o`.

Also, it is generally safer to put positional arguments (here: input.bam) last. That order is also how most tools are described in their documentation. (It happens to work here for samtools, so it’s mainly for consistency here.)